packages/appstore/src/components/wallet-transfer/wallet-transfer.tsx

```diff
@@ -215,6 +203,8 @@ const WalletTransfer = observer(({ is_wallet_name_visible, setIsWalletNameVisibl
                                 </Field>
                                 <Divider />
                                 <TransferAccountSelector
+                                    //add key to reset the selector state when value updated
+                                    key={JSON.stringify(from_account)}
```

Comment:
add a key here in order to not use useEffect inside TransferAccountSelector when we update the value (account), refer [here](https://github.com/binary-com/deriv-app/pull/9673/files#r1291081971)